src/constants/deploy-env.ts

```diff
@@ -0,0 +1,10 @@
+export const LOCAL = 'local'
```

Comment:
What does `local` mean here? How is it different from `dev`?

Reply:
I'm not sure to be honest, but this is my interpretation:
- local: the instance on your work computer/laptop
- dev: an instance in the cloud that runs the up-to-date code from master
- staging: an instance in the cloud that runs the latest release for QA
- production: an instance in the cloud that runs the latest after it's passed QA in staging

Reply:
That's my understanding too.

We currently use `dev` and `staging` roughly interchangeably in our current infrastructure setup, but that's something that's liable (hopefully!) to change. We could eventually collapse dev and staging into a single environment once we get close to true continuous delivery.